show-version-diff/README.md

```diff
@@ -0,0 +1,9 @@
+# Show Version Diff
+
+Show the changes between all versions of a single file using `diff`.
+
+The following command shows all changes to `deck.md`, ignoring the `pre-1.7` folder.
```

Comment:
```suggestion
Run the diff script from the `show-version-diff` folder. The following 
command shows all changes to `deck.md`, ignoring the `pre-1.7` folder:
```
Useful to mention that you have to be in this folder to run it. I ran into that issue on my first attempt to test. Seems like common sense in hindsight, but it wasn't my first instinct.